governance/default.go

```diff
@@ -146,12 +146,16 @@ type VoteStatus struct {
 	Num    uint64      `json:"num"`
 }
 
+type VoteSet struct {
+	items map[string]VoteStatus
+	mu    *sync.RWMutex
+}
+
 type Governance struct {
 	ChainConfig *params.ChainConfig
 
 	// Map used to keep multiple types of votes
-	voteMap     map[string]VoteStatus
-	voteMapLock sync.RWMutex
+	voteMap VoteSet
```

Comment:
Variable name is still `voteMap` but the type has been changed to `VoteSet`.
It would be better to use `VoteMap` instead of `VoteSet` since it internally manages a map.

Reply:
I think `voteMap` is better than `voteSet`.

Reply:
@ehnuje @KimKyungup Thanks for your feedback. I changed the name of the type as `VoteMap`. PTAL =)